src/lib/remark-plugins/rewrite-static-assets.ts

```diff
@@ -15,7 +15,11 @@ import { Plugin } from 'unified'
 import { Node } from 'unist'
 import { Image, Definition } from 'mdast'
 
-const ASSET_API_ENDPOINT = `${process.env.MKTG_CONTENT_API}/api/assets`
+// This env is set for local docker previews by a custom asset server,
+// otherwise we use the content api for previews / prod
+const ASSET_API_ENDPOINT =
+	process.env.ASSET_API_ENDPOINT ||
```

Comment:
I'm not 100% sure, but I think we may need to include `ASSET_API_ENDPOINT` in our `next.config.js` file?

Reply:
Why is that?

Reply:
We do it for other custom environment variables including the `MKTG_CONTENT_API` one, and there are some brief docs detailing it: https://nextjs.org/docs/api-reference/next.config.js/environment-variables. But I'm not sure how to test to verify whether or not it's needed. 🤔

Reply:
I tested this locally with the tutorials repo and it seemed to work properly. I can make a pre-release of the image and test it to 100% verify

Reply:
But i'm also curious to learn more about the envs in next.config!? 🤔

Reply:
I added it to next.config to be safe!